### PR TITLE
feat: add support for `first` summary option for `define_metric`

### LINKED ISSUE #9936: [Feature]: Add 'first' to define_metric summary options
### Description

Sometimes (e.g. in finetuning) it's useful to compare performance at step 0 (pretrained model) with later steps. A First summary metric would support this

### Suggested Solution

Add first to the list of supported summaries in define_metric

### PR DESCRIPTION
**Pull request by kptkin**
Description
-----------

- Fixes WB-25427
- Fixes #9936 

What does the PR do? Include a concise description of the PR contents.

see PR title

- [x] I updated CHANGELOG.unreleased.md, or it's not applicable

Testing
-------
How was this PR tested?

### CODE REVIEW

**Review by dmitryduev — DISMISSED** [review]

File: CHANGELOG.unreleased.md [hunk 1]
```diff
@@ -14,5 +14,8 @@ Section headings should be at level 3 (e.g. `### Added`).
 ## Unreleased
 
 ### Fixed
-- Correct the artifact url for organization registry artifacts to be independent of the artifact type. (@ibindlish in https://github.com/wandb/wandb/pull/10049)
-- Suffixes on sanitized `InternalArtifact` names have been shortened to 6 alphanumeric characters. (@tonyyli-wandb in https://github.com/wandb/wandb/pull/10102)
+- Correct the artifact url for organization registry artifacts to be independent of the artifact type (@ibindlish in https://github.com/wandb/wandb/pull/10049)
+- Suffixes on sanitized `InternalArtifact` names have been shortened to 6 alphanumeric characters (@tonyyli-wandb in https://github.com/wandb/wandb/pull/10102)
+
+### Added
+- Support `first` summary option in `define_metric` (@kptkin in https://github.com/wandb/wandb/pull/10121)
```
Comment (timoffex) [inline CHANGELOG.unreleased.md]: Could you move this above Fixed to match the usual order?

**Review by timoffex — APPROVED** [review]

**Review by timoffex — APPROVED** [review]
